AsyncDisplayKit/ASEditableTextNode.mm

```diff
@@ -133,7 +135,7 @@ - (void)didLoad
   [self.view addSubview:_placeholderTextKitComponents.textView];
 
   // Create and configure our text view.
-  _textKitComponents.textView = [[_ASDisabledPanUITextView alloc] initWithFrame:CGRectZero textContainer:_textKitComponents.textContainer];
+  _textKitComponents.textView = _textView;
```

Comment:
Since we're doing this, it looks like we might as well not add the _textView instance variable at all.  Let's just use _textKitComponents.textView, anywhere and everywhere that _textView would otherwise need to be used.  This ensures that we wouldn't have a case where only one of them is initialized but they aren't set up to be the same.